File: beecrowd/1_Beginner/1079_Weighted_Averages.c
#include <stdio.h>
#include <math.h>

int main()
{
    int N;
    scanf("%d", &N);

    float A;
    float B;
    float C;
    float output[N];

    for(int i=0; i<N; i++)
    {
        float avg=0.0;

        scanf("%f",&A);
        scanf("%f",&B);
        scanf("%f",&C);

        avg = ((A*2) + (B*3) + (C*5))/10;
        output[i]=avg;
    }

    for(int i=0; i<N; i++){
        printf("%.1f\n",output[i]);
    }
    return 0;
}

File: beecrowd/1_Beginner/Python/1079_Weighted_Averages.py
n = int(input())  # read the first line as integer
lines = []  # create an empty list to store the linesay
output = []

for i in range(n):
    row = list(map(float, input().split()))  # read each row and convert to float
    lines.append(row)  # add the row to the linesay

for line in lines:
    avg = ((line[0]*2) + (line[1]*3) + (line[2]*5))/10;
    output.append(avg)

for out in output:
    format_float = "{:.1f}".format(out)
    print(format_float)